Remove to be scheduled column from bugs board (#14679)
fleetdm/fleet: handbook/product/README.md

Before assigning an engineering manager to [estimate](https://fleetdm.com/handbook/engineering#sprint-ceremonies) a user story, the product designer ensures the product section of the user story [checklist](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=story&projects=&template=story.md&title=) is complete. 

> Once a bug has gone through design and is considered "Settled", the designer removes the `:product` label and moves the issue to the 'Sprint backlog' column on the "Bugs" board. The product manager then prioritizes the bug into the "Sprint backlog" and assigns the group engineering manager. 

Reviewer: ```suggestion
Once a bug has gone through design and is considered "Settled", the designer removes the `:product` label and moves the issue to the 'Sprint backlog' column on the "Bugs" board and assigns the group engineering manager. 
```

I think we can cut the bit about PM prioritizing the bug. Product designer can just assign the EM

If a bug is unreleased or [critical](https://fleetdm.com/handbook/engineering#critical-bugs), it is addressed in the current sprint. Otherwise, it may be prioritized and estimated for the next sprint. If a bug [requires drafting](https://fleetdm.com/handbook/engineering#in-product-drafting-as-needed) to determine the expected functionality, the bug should undergo [expedited drafting](#expedited-drafting). 

> If a bug is not addressed within six weeks, it is [sent to the product team for triage](https://fleetdm.com/handbook/engineering#in-engineering). Each sprint, the Head of Product design reviews these bugs. Bugs are categorized as follows:

Reviewer: ```suggestion
If a bug is not addressed within six weeks, it is [sent to the product team for triage](https://fleetdm.com/handbook/engineering#in-engineering). Each sprint, the Director of Product Development reviews these bugs. Bugs are categorized as follows:
```

@lukeheath after reading through the age out process, maybe this should be you? 

This way, the entire process is owned and driven by one person (less handoffs).

Reviewer: cc @mikermcneil

Reviewer: UPDATE: I spoke to Mike. We decided that Product (Noah) will own reviewing churned bugs to prepare for "Churned bug review" call. Noah and @lukeheath will meet at "Churned bug review" to align on which bugs we will continue to work on.

(noahtalerman)

- **Schedule**: the bug should be prioritized in the next sprint if there's engineering capacity for it.
- **De-prioritized**: the issue will be closed and the necessary subsequent steps will be initiated. This might include updating documentation and informing the community.

The Head of Product Design meets with the CEO to discuss and finalize the outcomes for the churned bugs.

Once outcomes have been approved by the CEO, The Head of Product Design will complete the churned bug clean-up ritual. Below are the steps for each category:

> - **Schedule**: Product Operations should remove the `:product` label, move the bug ticket to the 'Sprint backlog' column on the bug board, and assign it to the appropriate group's Engineering Manager so that it can be prioritized into the sprint backlog.

Reviewer: ```suggestion
- **Schedule**: Remove the `:product` label, move the bug ticket to the 'Sprint backlog' column on the bug board, and assign it to the appropriate group's Engineering Manager so that it can be prioritized into the sprint backlog.
```
